### PR TITLE
ZEPPELIN-3202. Added missing test dependencies in the scio module

### PR DESCRIPTION
**Pull request by HorizonNet**
### What is this PR for?

Added missing test dependencies for the **scio** module to prevent test failures when running `mvn clean install`.

### What type of PR is it?
Bug Fix

### Todos
* [ ] - Task

### What is the Jira issue?
* https://issues.apache.org/jira/browse/ZEPPELIN-3202

### How should this be tested?
* CI pass

### Questions:
* Does the licenses files need update? no
* Is there breaking changes for older versions? no
* Does this needs documentation? no

### CODE REVIEW

File: scio/pom.xml [hunk 1]
```diff
@@ -43,6 +43,9 @@
     <plugin.shade.version>2.3</plugin.shade.version>
     <plugin.scala.version>2.15.2</plugin.scala.version>
     <plugin.avro.version>1.7.7</plugin.avro.version>
+
+    <!--test library versions-->
+    <hamcrest.all.version>1.3</hamcrest.all.version>
```
Comment (felixcheung) [inline scio/pom.xml]: see https://github.com/apache/zeppelin/pull/2759/files#diff-608f20f45ac75cfdb867d8d83eb0216bR98?

Comment (felixcheung) [reply scio/pom.xml]: this should be added as test scope

Comment (HorizonNet) [reply scio/pom.xml]: **hamcrest-all** is already set as test scope (see line 107ff).

**Comment by HorizonNet** [comment]
I kicked off another Travis build and it seems that these failing tests are flakey.

**Comment by HorizonNet** [comment]
I just reran the failing builds, as suggested by @zjffdu in ZEPPELIN-3182, and all are green now.

**Comment by felixcheung** [comment]
merging if no more comment

**Pull request closed without merging**